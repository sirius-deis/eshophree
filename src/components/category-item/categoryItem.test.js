import { render, screen } from "@testing-library/react";
import CategoryItem from "./categoryItem";


describe("CategoryItem component", () => {
  it("should match snapshot", function () {
    const { container } = render(<CategoryItem />);
    expect(container).toMatchSnapshot();
  });
  it("should render a category name", () => {
    render(<CategoryItem>Electronics</CategoryItem>);
    expect(screen.getByText("Electronics")).toBeInTheDocument();
  });
})