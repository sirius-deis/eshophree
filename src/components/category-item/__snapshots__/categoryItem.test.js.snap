// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CategoryItem component should match snapshot 1`] = `
<div>
  <div
    class="sc-bgqQcB jAzBDU"
  />
</div>
`;
